pkgs/by-name/sv/svg-term/package.nix

```
@@ -0,0 +1,55 @@
+{
+  asciinema,
+  fetchFromGitHub,
+  fetchYarnDeps,
+  lib,
+  makeWrapper,
+  nodejs,
+  stdenv,
+  versionCheckHook,
+  yarnBuildHook,
+  yarnConfigHook,
+  yarnInstallHook,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "svg-term";
+  version = "2.1.1";
+
+  src = fetchFromGitHub {
+    owner = "marionebl";
+    repo = "svg-term-cli";
+    tag = "v${finalAttrs.version}";
+    hash = "sha256-sB4/SM48UmqaYKj6kzfjzITroL0l/QL4Gg5GSrQ+pdk=";
+  };
+
+  yarnOfflineCache = fetchYarnDeps {
+    yarnLock = "${finalAttrs.src}/yarn.lock";
```

```suggestion
    inherit (finalAttrs) src;
```

Works the same and is a bit cleaner imo, feel free to ignore